app/assets/stylesheets/application.scss

```diff
@@ -49,12 +49,12 @@ h3.quote:before, h3.quote:after {
 
 .mdl-card__title {
   font-size: 13px;
-  margin-top: 20px;
+  padding: 0px;
 }
 
 .group { 
   position: relative; 
-  margin-bottom: 45px; 
+  padding: 25px 0px 25px 0px; 
```

Comment:
Shorthand form for property `padding` should be written more concisely as `25px 0px` instead of `25px 0px 25px 0px`<br>Line contains trailing whitespace<br>`0px` should be written without units as `0`